build-lib/lib.js

```diff
@@ -95,12 +89,13 @@ window.ShopifyBuy = require('shopify-buy/shopify').default;
     const minifiedTree = uglifyJavaScript(funnel(tree, {
       getDestinationPath: function (path) {
         return path.replace(/\.js/, '.min.js');
-      }
+      },
+      exclude: ['**/*.map']
     }));
 
-    const concatenatedScripts = new Licenser([mergeTrees([tree, minifiedTree])]);
+    const licensedScripts = new Licenser([mergeTrees([tree, minifiedTree], { overwrite : true })]);
 
-    tree = mergeTrees([concatenatedScripts, nodeLibOutput]);
+    tree = mergeTrees([tree, minifiedTree, licensedScripts, nodeLibOutput], { overwrite : true });
```

Comment:
What was the intent of this change? Getting the thrown error on overwrite indicates that your output trees collide somewhere.